_haikus/Dead_Dev.md

```diff
@@ -0,0 +1,9 @@
+---
+layout: haiku
+title: Dead_Dev
+author: goshrow
+---
+
+Hi Dev! Your system<br>
+Is running low as your age<br> 
+Yet your IP lives on<br>
```

Comment:
This is one syllable too many as `IP` is two syllables. Maybe you can try `Yet your IP lives` or `Your IP lives on`.